examples/Generic/README.md

```diff
@@ -2,21 +2,160 @@
 
 This directory contains a minimal example that shows how to use `AstarteGenericDevice`.
 
-The device is instantiated in `ExampleDevice.java`, the code is commented to highlight all the
-things needeed to create your Astarte Device.
+The device is instantiated in `ExampleDevice`, the code is commented to highlight all the
+things needed to create your Astarte Device.
+
+
+## Downloading
+
+Add to "dependencies" (as for any module you need):
```

Comment:
`(as for any module you need)` this information is useless and it clutters the text, let's remove it.